var __v="\u003cpre class='k-pre'\u003e除了 調用 subscriber.error 外 之間 throw 也可以 通知 訂閱者 發生錯誤(者兩種方法 對訂閱者來說 沒有差別)\n當 throw 的堆棧信息 只有 Angular 內部框架的 數據而 毫無用處 但卻更慢 所以 推薦使用 error 而非 throw 通知錯誤\n\n如果 throw 回調函數 會 結束 Angular會幫你捕獲到此 異常 並通知 訂閱者 發送錯誤\n如果 使用 error 通知 回調函數 不會 結束 而是 繼續 error 後面的代碼 error 只不過時一個 再普通不過的 函數調用\n\n通常 當訂閱者 退訂時 你需要 停止邏輯 可以在 回調函數中 返回一個 帶 unsubscribe 方法的 對象\n退訂時 此函數 會被回調\n\n即使 訂閱者 已經 退訂 你依然 可以調用 next error complete 但 這些函數會 直接返回 訂閱者 \n\u003c/pre\u003e"